tests/cases/conformance/es2017/useStringPadding3.ts

```diff
@@ -0,0 +1,7 @@
+// @target: es6
```

Comment:
the tests need baselines, so travis is going to fail. you need to run `jake baseline-accept` after running the tests to include the baselines.

For this change, i do not think you need the tests. so i would leave them out.